src/components/calculator.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable react/prop-types, react/prefer-stateless-function */
+import { Component } from 'react';
+import './calculator.css';
+
+class Calculator extends Component {
+  render() {
+    const pureFunction = this.props;
+    console.log(pureFunction);
+    return (
```

Comment:
- [ ] Please don't disable eslint errors, kindly fix them without disabling them,

- [ ] Remove the `console.log` in the code that has been pushed.

- [ ]  Instead of repeating yourself ( `className=" button flex-child"`),  I would recommend creating a div that will hold a className as well as style it without repeating. Prevent DRY